addons/mail/static/tests/tours/discuss_configuration_tour.js

```diff
@@ -47,5 +47,58 @@ registry.category("web_tour.tours").add("discuss_configuration_tour", {
             content: _t("Click to close"),
             run: "click",
         },
+        {
+            trigger: ".o_main_navbar button:contains('Configuration')",
+            content: _t("Click to see the setting options"),
+            run: "click",
+        },
+        {
+            trigger: ".dropdown-menu a:contains('Voice & Video')",
+            content: _t("Open the Voice & Video settings"),
+            run: "click",
+        },
+        {
+            trigger: "select[name='inputDevice']",
+            content: _t("Select the input device"),
+        },
+        {
+            trigger: "button:contains('Voice Detection')",
+            content: _t("Click to enable voice detection"),
+            run: "click",
+        },
+        {
+            trigger: "label:contains('Voice detection threshold')",
+            content: _t("Adjust the voice detection threshold"),
+        },
+        {
+            trigger: "button:contains('Push to Talk')",
+            content: _t("Click to enable push to talk"),
+            run: "click",
+        },
+        {
+            trigger: "label:contains('Push-to-talk key')",
+            content: _t("Click to register a new key"),
+        },
+        {
+            trigger: "label:contains('Delay after releasing push-to-talk')",
+            content: _t("Adjust the delay after releasing push-to-talk"),
+        },
+        {
+            trigger: "input[aria-label='Show video participants only']",
+            content: _t("Click to show only video participants"),
+        },
+        {
+            trigger: "input[aria-label='Blur video background']",
+            content: _t("Click to blur the video background"),
+            run: "click",
+        },
+        {
+            trigger: "label:contains('Background blur intensity')",
+            content: _t("Adjust the background blur intensity"),
+        },
+        {
+            trigger: "label:contains('Edge blur intensity')",
+            content: _t("Adjust the edge blur intensity"),
+        },
```

Comment:
The reason why we have those content is to point out what is happening each step. When having an issue, we have a clue to find out what is the expected behavior. I personally consider these as a kind of commments.

Reply:
The less comments the better. The selector are expressive enough. In general if you feel the need for `content` then the actual issue is that `trigger` is not explicit enough and should be adapted.

The `content` is useful for non-test tours. But for test tours this adds more LOCS, reducing code maintenance, when this has barely any value. I much prefer having a grasp of the whole test fast rather than rely on this `content` when debugging. Also a code comment can be a lie, so that's another reason to cautiously add them and sometimes just avoid comments.